### PR TITLE
Add refresh pooling with active and inactive simulations

### PR DESCRIPTION
**Pull request by ramindu90**

### CODE REVIEW

File: components/org.wso2.carbon.event.simulator.core/src/main/java/org/wso2/carbon/event/simulator/core/service/EventSimulatorMap.java [hunk 1]
```diff
@@ -32,15 +32,17 @@
...
 
 /**
  * EventSimulatorMap holds the simulators available
  */
 public class EventSimulatorMap {
     private static final Logger log = LoggerFactory.getLogger(EventSimulatorMap.class);
     private static final EventSimulatorMap instance = new EventSimulatorMap();
-    private final HashMap<String, ActiveSimulatorData> activeSimulatorMap = new HashMap<>();
-    private final HashMap<String, ResourceDependencyData> inActiveSimulatorMap = new HashMap<>();
+    private final Map<String, ActiveSimulatorData> activeSimulatorMap = new ConcurrentHashMap<>();
```
Comment (nirmal070125) [inline components/org.wso2.carbon.event.simulator.core/src/main/java/org/wso2/carbon/event/simulator/core/service/EventSimulatorMap.java]: What's the key of the map? better to comment them.

File: components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/event-simulator/feed_simulator.js [hunk 2]
```diff
@@ -292,6 +302,30 @@ define(['jquery', 'log', './simulator-rest-client', 'lodash', /* void libs */'bo
...
+                    // var consoleOptions = {};
+                    // var options = {};
+                    // _.set(options, '_type', "CONSOLE");
+                    // _.set(options, 'title', "Console");
+                    // _.set(options, 'currentFocusedFile', simulationName);
+                    // _.set(options, 'statusForCurrentFocusedFile', "simulation");
+                    // _.set(options, 'message', "Started Successfully!");
+                    // _.set(consoleOptions, 'consoleOptions', options);
+                    // consoleListManager.newConsole(consoleOptions);
+                    // consoleListManager.newConsole(consoleOptions);
```
Comment (nirmal070125) [inline components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/js/event-simulator/feed_simulator.js]: Why keeping the commented code?

**Comment by ramindu90** [comment]
@nirmal070125 added the changes.

File: components/org.wso2.carbon.event.simulator.core/src/main/java/org/wso2/carbon/event/simulator/core/service/EventSimulatorMap.java (hunk 1 above)
Comment (nirmal070125) [inline components/org.wso2.carbon.event.simulator.core/src/main/java/org/wso2/carbon/event/simulator/core/service/EventSimulatorMap.java]: eg: 
  /**
     * Key - name of the model
     * Value - @{@link PerceptronModel}
     */
    private Map<String, PerceptronModel> perceptronModelMap = new HashMap();

**Comment by ramindu90** [comment]
@nirmal070125 added the requested change

**Pull request merged**